signetdev/host/signetdev_linux.c

```diff
@@ -385,14 +386,20 @@ void *transaction_thread(void *arg)
 int signetdev_priv_issue_command(int command, void *p)
 {
 	intptr_t v[2] = {command, (intptr_t)p};
-	write(g_command_pipe[1], v, sizeof(intptr_t) * 2);
+	if (write(g_command_pipe[1], v, sizeof(intptr_t) * 2) == -1) {
+		perror("signetdev_priv_issue_command write error");
+	}
 	char cmd_resp;
-	read(g_command_resp_pipe[0], &cmd_resp, 1);
```

Comment:
This should return the error to caller like the others but need to make sure that the return value is differentiated from values read from the pipe

Reply:
I'd suggest -1. 
Or define an arbitrary positive number different than:
```
#define SIGNETDEV_PRIV_GET_RESP 1
#define SIGNETDEV_PRIV_NO_RESP 0
```
These are read from the pipe, right?

Reply:
A negative might be the right solution but lets punt and make this commit about doing the minimum to fix the warnings so long is there is a "//TODO" comment letting us know something needs to be done later.